include/base/libmesh_exceptions.h

```diff
@@ -65,7 +65,8 @@ class NotImplemented : public std::logic_error
 class FileError : public std::runtime_error
 {
 public:
-  FileError(const std::string & filename) : std::runtime_error( "Error accessing file: " + filename ) {}
+  FileError(const std::string & filename, const std::string msg="") :
+    std::runtime_error("Error with file `" + filename + "'\n" + std::move(msg)) {}
```

Comment:
Looks like an erroneous backtick here?

Reply:
Nah, that's deliberate - see the removed code in `libmesh_file_error_msg`.  There's a "matching" neutral-tick on the other side; the intention is to try to be clearer about files/paths with spaces in them.

Reply:
:+1: